src/renderer/components/cluster-icon/cluster-icon.scss

```diff
@@ -7,23 +7,25 @@
   user-select: none;
   cursor: pointer;
 
-  &.interactive {
-    img {
-      opacity: .55;
-    }
+
+  div.MuiAvatar-colorDefault {
+    font-weight:500;
+    text-transform: uppercase;
   }
 
-  &.active, &.interactive:hover {
-    background-color: #fff;
+  div.active {
+    background-color: var(--primary);
+  }
 
-    img {
-      opacity: 1;
-    }
+  div.default {
+    background-color: var(--halfGray);
   }
 
-  img {
-    width: var(--size);
-    height: var(--size);
```

Comment:
you may need to keep img styling for when user sets their own icon. Unless we are eliminating that?

Reply:
Oh right, I probably should revert these.